platforms/jvm/code-quality/src/main/java/org/gradle/api/plugins/quality/PmdExtension.java

```diff
@@ -237,7 +237,7 @@ public void setConsoleOutput(boolean consoleOutput) {
     /**
      * Controls whether to use incremental analysis or not.
      *
-     * This is only supported for PMD 6.0.0 or better. See <a href="https://docs.pmd-code.org/pmd-doc-7.13.0/pmd_userdocs_incremental_analysis.html"></a> for more details.
+     * This is only supported for PMD 6.0.0 or better. See <a href="https://docs.pmd-code.org/pmd-doc-7.13.0/pmd_userdocs_incremental_analysis.html">official documentation</a> for more details.
```

Comment:
https://docs.gradle.org/8.14.1/javadoc/org/gradle/api/plugins/quality/PmdExtension.html#getIncrementalAnalysis()

![image](https://github.com/user-attachments/assets/573d4ffb-e2dc-4077-9862-7f3875a324ef)

Missing text for `<a>` element